inductiva/tasks/methods.py

```diff
@@ -62,12 +123,12 @@ def list(num_tasks,
     Example usage:
         # list the last 5 tasks that were successful
         inductiva.tasks.list(5, status="success")
-        ID                   Simulator    Status       Submitted            Started              Duration     VM Type
-        1691150776862178362  openfoam     success      04 Aug, 12:06:17     04 Aug, 12:06:18     0h 1m 53s    c2-standard-8
-        1691149904961476240  openfoam     success      04 Aug, 11:51:46     04 Aug, 11:51:46     0h 1m 28s    c2-standard-8
-        1691081881158823776  openfoam     success      03 Aug, 16:58:02     03 Aug, 16:58:02     0h 1m 20s    n2-standard-32
-        1691081409916619414  openfoam     success      03 Aug, 16:50:11     03 Aug, 16:50:11     0h 1m 20s    n2-standard-32
-        1691080520213617518  openfoam     success      03 Aug, 16:35:21     03 Aug, 16:35:21     0h 1m 23s    n2-standard-32
+                         ID       Simulator               Status            Submitted              Started        Duration            VM Type
+        1691150776862178362        openfoam              success     04 Aug, 12:06:17     04 Aug, 12:06:18       0h 1m 53s      c2-standard-8
+        1691149904961476240        openfoam              success     04 Aug, 11:51:46     04 Aug, 11:51:46       0h 1m 28s      c2-standard-8
+        1691081881158823776        openfoam              success     03 Aug, 16:58:02     03 Aug, 16:58:02       0h 1m 20s     n2-standard-32
+        1691081409916619414        openfoam              success     03 Aug, 16:50:11     03 Aug, 16:50:11       0h 1m 20s     n2-standard-32
+        1691080520213617518        openfoam              success     03 Aug, 16:35:21     03 Aug, 16:35:21       0h 1m 23s     n2-standard-32
 
     Args:
         num_tasks: The number of tasks to list.
```

Comment:
The comment should say (if I understand correctly that these are the "the most recent num tasks" (btw, in terms of what, launch time?) that respect the status filter... Right?

Reply:
If filtering criteria is provided, this will get the most recent N tasks that match that criteria. I'll change the docstring to clarify that